docs/src/man/hosting.md

```diff
@@ -227,15 +231,9 @@ GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 to the configuration file, as showed in the [previous section](@ref GitHub-Actions).
 
 !!! note
-    You can only use `GITHUB_TOKEN` for authentication if the target repository
-    of the deployment is the same as the current repository. In order to push
-    elsewhere you should instead use a SSH deploy key.
-
-!!! warning "GitHub Pages and GitHub Token"
-    Currently the GitHub Page build is not triggered when the GitHub provided
-    `GITHUB_TOKEN` is used for authentication. See
-    [issue #1177](https://github.com/JuliaDocs/Documenter.jl/issues/1177)
-    for more information.
+    While GitHub will automatically create a github token for you named "GITHUB_TOKEN",
+    it will not have push access. Therefore, you must override
+    GitHub by making a new personal access token with the same name.
```

Comment:
It should specifically say what to do.

Reply:
I can add a note like `(see instructions above)` because I did add instructions to do this?

Reply:
Yes, and a link to that part of the instructions